modules/eco-configuring-machine-health-check-with-self-node-remediation.adoc

```diff
@@ -15,30 +15,30 @@ Use the following procedure to configure the machine health checks to use the Po
 
 .Procedure
 
-. Create a `PoisonPillRemediationTemplate` CR:
+. Create a `SelfNodeRemediationTemplate` CR:
 
-.. Define the `PoisonPillRemediationTemplate` CR:
+.. Define the `SelfNodeRemediationTemplate` CR:
 +
 [source,yaml]
 ----
-apiVersion: poison-pill.medik8s.io/v1alpha1
-kind: PoisonPillRemediationTemplate
+apiVersion: self-node-remediation.medik8s.io/v1alpha1
+kind: SelfNodeRemediationTemplate
 metadata:
   namespace: openshift-machine-api
-  name: poisonpillremediationtemplate-sample
+  name: <self-node-remediation-template>
 spec:
   template:
     spec: {}
 ----
 
-.. To create the `PoisonPillRemediationTemplate` CR, run the following command:
+.. To create the `SelfNodeRemediationTemplate` CR, run the following command:
 +
 [source,terminal]
 ----
-$ oc create -f <ppr-name>.yaml
+$ oc create -f <snr-name>.yaml
 ----
 
-. Create or update the `MachineHealthCheck` CR to point to the `PoisonPillRemediationTemplate` CR:
+. Create or update the `MachineHealthCheck` CR to point to the `SelfNodeRemediationTemplate` CR:
```

Comment:
Meantime, you can't edit in 4.11 I don't see this section. Seems like a bug (I'll open)